### PR TITLE
homework number 12

### PR DESCRIPTION
**Pull request by vaasilik**
changed config for link 
added 1 folder 2-Locators-Actions
added 3 test cases

### CODE REVIEW

File: cypress/e2e/2-Locators-Actions/forms-layouts.cy.js
Comment (landriiuk) [inline cypress/e2e/2-Locators-Actions/forms-layouts.cy.js:1]: a good approach to use within here!
Pls, on step about POM don't forget to delete this comments

File: cypress/e2e/2-Locators-Actions/forms-layouts.cy.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+describe("Тест 3", () => {
```
Comment (landriiuk) [inline cypress/e2e/2-Locators-Actions/forms-layouts.cy.js]:
```suggestion
describe("Verify layouts page", () => {
```

File: cypress/e2e/2-Locators-Actions/forms-layouts.cy.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    it("Сторінка /pages/forms/layouts", () => {
```
Comment (landriiuk) [inline cypress/e2e/2-Locators-Actions/forms-layouts.cy.js]: Pls change test name to english

File: cypress/e2e/2-Locators-Actions/layout.cy.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+describe("Тест 1", () => {
+    it("Сторінка /pages/layout/stepper", () => {
```
Comment (landriiuk) [inline cypress/e2e/2-Locators-Actions/layout.cy.js]: description and file name should be written in English

File: cypress/e2e/2-Locators-Actions/modal-overlays.cy.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
+describe("Тест 2", () => {
+    it("Сторінка /pages/modal-overlays/dialog", () => {
```
Comment (landriiuk) [inline cypress/e2e/2-Locators-Actions/modal-overlays.cy.js]: description and file name should be written in English

File: cypress/e2e/2-Locators-Actions/layout.cy.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+      cy.visit("/pages/layout/stepper");
+  
+      // Функція для перевірки тексту та натискання кнопки "NEXT"
+      function checkStepText(stepText) {
+        cy.get(".step-content").should("contain", stepText);
+        cy.get("body > ngx-app > ngx-pages > ngx-one-column-layout > nb-layout > div > div > div > div > div > nb-layout-column > ngx-components > ngx-stepper > nb-card > nb-card-body > nb-stepper > div.step-content > button:nth-child(3)").click();
```
Comment (landriiuk) [inline cypress/e2e/2-Locators-Actions/layout.cy.js]: there is a better selector for it : 
```suggestion
        cy.get("h3:eq(0)").click();
```

File: cypress/e2e/2-Locators-Actions/layout.cy.js [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+      }
```
Comment (landriiuk) [inline cypress/e2e/2-Locators-Actions/layout.cy.js]:
```suggestion
      function checkStepText(stepText) {
        cy.get(".step-content").should("contain", stepText);
        stepText === "Step content #4"
        ?"Skip"
       : cy.get("h3:eq(0)").click();
      }
```

File: cypress/e2e/2-Locators-Actions/layout.cy.js [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+  
+      // Перевірка тексту на кожному стеці та натискання кнопки "NEXT"
+      checkStepText("Step content #1");
+      checkStepText("Step content #2");
+      checkStepText("Step content #3");
+  
+      // На останньому стеці кнопка "NEXT" не активна, тому її не перевіряємо та не натискаємо
+      cy.get(".step-content").should("contain", "Step content #4");
```
Comment (landriiuk) [inline cypress/e2e/2-Locators-Actions/layout.cy.js]: you can add additional verification into your checkStepText() method.  Added example 
```suggestion
      checkStepText("Step content #4");
```

File: cypress/e2e/2-Locators-Actions/modal-overlays.cy.js [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+      // Відкриваємо сторінку
+      cy.visit("/pages/modal-overlays/dialog");
+  
+      // Натискаємо на кнопку, що викликає модальне вікно
+      cy.get("body > ngx-app > ngx-pages > ngx-one-column-layout > nb-layout > div.scrollable-container > div > div > div > div > nb-layout-column > ngx-modal-overlays > ngx-dialog > div > div:nth-child(5) > nb-card > nb-card-body > button").click();
```
Comment (landriiuk) [inline cypress/e2e/2-Locators-Actions/modal-overlays.cy.js]: you can use here 
button:contains("Enter Name")
or cy.contains("Enter Name")

**Review by landriiuk — CHANGES_REQUESTED** [review]
Added commands needed to improve your code
Pls check them

File: cypress/e2e/2-Locators-Actions/layout.cy.js (hunk 5 above)
Comment (vaasilik) [reply cypress/e2e/2-Locators-Actions/layout.cy.js]: для цього тесту в мене селектор, який ти вказала - не працює
і відповідно всі інші зміни, додані до цього тесту - фейляться

**Pull request closed without merging**